src/components/PrintModal/WatermarkModal/WatermarkModal.js

```diff
@@ -310,7 +327,18 @@ class WatermarkModal extends React.PureComponent {
 
               </div>
               <div className="form-field">
-
+                <label>
+                  {t(`option.watermark.font`)}
+                </label>
+                <select
```

Comment:
If an element has `data-element` then users should be able to disable it by using `instance.disableElements`. You can either hook up with redux or just remove `data-element` if you don't want users to be able to disable it.

Reply:
I don't think I want users to be able to disable it.
However, I noticed that the end to end tests I wrote a while back look for some of these data elements so that I can fill out the form properly.

What would be your suggestion for this problem?
I'm leaning towards keeping data element? I think the user should know what he/she is doing is he/she truly wants to disable these watermark form elements?

Reply:
If you don't want users to be able to disable it then you need to remove `data-element`. Otherwise it may confuse them if they are following this https://www.pdftron.com/documentation/web/guides/hiding-elements guide.

I think for now you can give it an id or a class name and change cypress to use these attributes. Just to make sure that we are on the same page, when I say users I mean developers, not end users...

Reply:
done. for some elements that are react components, i left them as "dataElements"